sdk/eventhub/azure-eventhubs/README.md

```diff
@@ -269,69 +224,62 @@ import asyncio
+a consumer group in an Event Hub instance.The `EventHubConsumerClient` uses an instance of `CheckpointStore` to update checkpoints
 and to store the relevant information required by the load balancing algorithm.
 
 Search pypi with the prefix `azure-eventhub-checkpointstore` to
-find packages that support this and use the PartitionManager implementation from one such package. Please note that both sync and async libraries are provided.
+find packages that support this and use the `CheckpointStore` implementation from one such package. Please note that both sync and async libraries are provided.
 
 In the below example, we create an instance of `EventHubConsumerClient` and use a `BlobCheckpointStore`. You need
 to [create an Azure Storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal)
 and a [Blob Container](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container) to run the code.
 
-[Azure Blob Storage Partition Manager Async](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio)
+[Azure Blob Storage Checkpoint Store Async](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio)
 and [Azure Blob Storage Checkpoint Store Sync](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhubs-checkpointstoreblob)
-are one of the `PartitionManager` implementations we provide that applies Azure Blob Storage as the persistent store.
+are one of the `CheckpointStore` implementations we provide that applies Azure Blob Storage as the persistent store.
 
 
 ```python
 import asyncio
 
 from azure.eventhub.aio import EventHubConsumerClient
-from azure.storage.blob.aio import ContainerClient
 from azure.eventhub.extensions.checkpointstoreblobaio import BlobCheckpointStore
 
-RECEIVE_TIMEOUT = 5  # timeout in seconds for a receiving operation. 0 or None means no timeout
-RETRY_TOTAL = 3  # max number of retries for receive operations within the receive timeout. Actual number of retries clould be less if RECEIVE_TIMEOUT is too small
 connection_str = '<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>'
-event_hub_path = '<< NAME OF THE EVENT HUB >>'
+eventhub_name = '<< NAME OF THE EVENT HUB >>'
 storage_connection_str = '<< CONNECTION STRING FOR THE STORAGE >>'
 container_name = '<<STRING FOR THE BLOB NAME>>'
 
-async def do_operation(event):
-    # do some sync or async operations. If the operation is i/o intensive, async will have better performance
-    print(event)
-
-
 async def process_event(partition_context, event):
-    await partition_context.update_checkpoint(event)
+    pass # do something
+    await partition_context.update_checkpoint(event)  # Or update_checkpoint every N events for better performance.
```

Comment:
Remove `pass` here - just have the comment `# do something`.

Reply:
Updated